tests/core_tests.py

```diff
@@ -531,7 +531,32 @@ def test_user_profile(self):
         data = self.get_json_resp('/superset/fave_dashboards/{}/'.format(userid))
         self.assertNotIn('message', data)
 
+    def test_import_csv(self):
+        self.login(username='admin')
+        config = app.config
+
+        test_file = open('tests/testCSV.csv', 'rb')
```

Comment:
From memory, I think there are issues with the native csv lib around unicode and py2<>py3 and had to use the external unicodecsv package to work around it.